packages/react-components/react-checkbox/etc/react-checkbox.api.md

```diff
@@ -8,31 +8,25 @@
 
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 19 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26416&commitId=dc25f851486a41e340994e05c61eabb7243174ee&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1756 screenshots unchanged, 0 screenshots with different dimensions and 19 screenshots with visible difference.


  <details><summary><b>unknown</b> 19 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ComboboxField converged.required.default.chromium.png|6| Changed|
|ComboboxField converged.size-large.default.chromium.png|1866| Changed|
|ComboboxField converged.size-small.default.chromium.png|2698| Changed|
|InputField converged.required.default.chromium.png|6| Changed|
|InputField converged.size-large.default.chromium.png|1791| Changed|
|InputField converged.size-small.default.chromium.png|2554| Changed|
|ProgressField converged.horizontal.default.chromium.png|12| Changed|
|ProgressField converged.required.default.chromium.png|6| Changed|
|ProgressField converged.validation.default.chromium.png|57| Changed|
|RadioGroupField converged.required.default.chromium.png|6| Changed|
|SelectField converged.required.default.chromium.png|6| Changed|
|SelectField converged.size-large.default.chromium.png|1899| Changed|
|SelectField converged.size-small.default.chromium.png|2639| Changed|
|SliderField converged.required.default.chromium.png|6| Changed|
|SpinButtonField converged.required.default.chromium.png|6| Changed|
|SwitchField converged.required.default.chromium.png|6| Changed|
|TextareaField converged.required.default.chromium.png|6| Changed|
|TextareaField converged.size-large.default.chromium.png|1785| Changed|
|TextareaField converged.size-small.default.chromium.png|2565| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>